src/api/app/views/webui/user/_dropdown_menu.html.haml

```diff
@@ -20,7 +20,12 @@
     %li
       = sprite_tag('cog_add')
       = link_to('Make user admin', user_update_path(user: { login: user.login, mark_as_admin: true }), |
-              method: :patch, data: { confirm: "Are you sure?" }) |
+                method: :patch, data: { confirm: "Are you sure?" }) |
+    - if ::Configuration.passwords_changable?(user)
```

Comment:
That would mean that you can also set a user to local user in Proxy Mode? That is not stated in the card. Do we want that?

Reply:
IIRC, we talked about a generic solution in the planning. But I can change it.